src/Compilers/CSharp/Portable/Emitter/Model/PropertySymbolAdapter.cs

```diff
@@ -14,32 +14,29 @@ internal partial class PropertySymbol :
     {
         #region IPropertyDefinition Members
 
-        IEnumerable<Cci.IMethodReference> Cci.IPropertyDefinition.Accessors
+        IEnumerable<Cci.IMethodReference> Cci.IPropertyDefinition.GetAccessors(EmitContext context)
         {
-            get
-            {
-                CheckDefinitionInvariant();
+            CheckDefinitionInvariant();
 
-                var getMethod = this.GetMethod;
-                if ((object)getMethod != null)
-                {
-                    yield return getMethod;
-                }
+            Cci.IMethodReference getMethod = this.GetMethod;
+            if (getMethod != null && Cci.Extensions.ShouldInclude(getMethod.GetResolvedMethod(context), context))
```

Comment:
Should `ShouldInclude(context)` begin with `if (context.IncludePrivateMembers) { return true; }`?